packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaEditor.integration.test.tsx

```diff
@@ -194,6 +237,26 @@ describe('TableColumnSchemaEditor', () => {
 
     // The default columns aren't editable, so we won't have a name text field.
     expect(screen.queryByLabelText('Name')).not.toBeInTheDocument()
+
+    await user.click(saveButton)
```

Comment:
For many of these existing tests, follow through with clicking "Save" and verify the calls are correct